hypebot/plugins/league/game_lib.py

```diff
@@ -453,6 +480,41 @@ def GetChampionImageUrl(self, champ_name):
       return None
     return self.GetImageUrl('champion', champ.image.full)
 
+  def GetReforgedRuneMessage(self, rune_name):
+    """Gets the reforged rune."""
+    if self._reforged_runes_name_complete:
+      rune = self._reforged_runes_name_complete.GuessThing(rune_name)
+      if rune:
+        return self._ComputeReforgedRuneCard(rune)
+
+    return 'Reforged rune "{}" not found.'.format(rune_name)
+
+  def _ComputeReforgedRuneCard(self, rune):
+    # Make runes more dank.
+    rune_name = util_lib.Dankify(rune.name)
+    tooltip = util_lib.Dankify(rune.long_desc)
+    var_table = client_vars.REFORGED_RUNE_VARS.get(rune.key, {})
+    tooltip = re.sub(
+        ur'@.+?@',
```

Comment:
Since we use "from __futures__ import  unicode_literals" We can pretend like python3 is the only thing in existence.